examples/instrumentation-quickstart/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,48 @@
+<!--
+ Copyright 2024 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ -->
+
+<Configuration status="WARN">
+  <Appenders>
+    <Console name="Console" target="SYSTEM_OUT">
+      <!-- [START opentelemetry_instrumentation_setup_logging] -->
```

Comment:
This is a different region tag than the docs currently use https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/97a8f20caf57263056ab0df8d8a54b4e7353eb2b/examples/instrumentation-quickstart/src/main/resources/logback.xml#L19

So I can update the docs before removing the logback.xml files